src/frontend/utils/utils.ts

```diff
@@ -1,3 +1,19 @@
 export const harIkkeVerdi = (str: string | undefined | null): boolean => !harVerdi(str);
 
 export const harVerdi = (str: string | undefined | null): boolean => !!str && str.trim() !== '';
+
+export const åpneFilIEgenTab = (
```

Comment:
Er dette nødvendig eller kunne man bare brukt `<Link target="_blank">`? Så den ble brukt på en knapp da så ikke alltid like kult å blande link og knapp 😢

Reply:
Man kan bruke link, men ulempene med dette tror jeg er at fanenavnet ikke blir satt til dokumenttittelen. Vi har fått tilbakemeldinger fra våre saksbehandlere på at man vil ha tabnavnet satt til dokumentittel for oversikt.

Reply:
Opprettet en task på å vurdere å endre andre links til å bruke denne metoden
https://favro.com/organization/98c34fb974ce445eac854de0/4d617346d79341c7fbd9a40a?card=NAV-20223